caep-interoperability-profile-1_0.md

```diff
@@ -0,0 +1,191 @@
+
+### Status Endpoint {#status-endpoint}
+The Transmitter Configuration Metadata MUST include the `status_endpoint` field. The specified endpoint MUST provide a way to Get and Update the Stream Status. The Transmitter MUST be able to pause and restart streams. For streams that are paused, the Transmitter MUST specify (offline) the resource constraints on how many events it can keep, or for how long. The way a Transmitter specifies this information is outside the scope of the SSF spec.
+
+### Verification Endpoint {#verification-endpoint}
+The Transmitter Configuration Metadata MUST include the `verification_endpoint` field. The specified endpoint MUST provide a way to request verification events to be sent.
+
+### Authorization Schemes
+The Transmitter Configuration Metadata MUST include the `authorization_schemes` field and its value MUST include the value
+
+~~~json
+{
+    "spec_urn": "urn:ietf:rfc:6749"
+}
+~~~
+
+### Streams {#common-stream-configuration}
+In all streams created by the Transmitter, the following MUST be true:
+
+#### Delivery {#common-delivery}
+The `delivery` field MUST be present in the Configuration of any Stream generated by the Transmitter, and its value MUST include the following:
+
+~~~ json
+{
+    "method" : "urn:ietf:rfc:8935"
+}
+~~~
+
+#### Stream Control
+The following Stream Configuration API Methods MUST be supported:
+
+**Creating a Stream**
+: Receivers MUST be able to create a Stream with the Transmitter using valid authorization with the Transmitter. The Transmitter MAY support multiple streams with the same Receiver
+
+**Reading Stream Configuration**
+: A Receiver MUST be able to obtain current Stream configuration from the Transmitter by providing a valid authorization
+
+**Getting the Stream Status**
+: A Receiver MUST be able to obtain the current Stream status from the Transmitter by providing a valid authorization
+
+**Stream Verification**
+: A Receiver MUST be able to verify the liveness of the Stream by requesting that the Transmitter send it a Stream Verificaiton event
+
+## Receivers {#common-receivers}
+Receivers MUST implement the following features:
+
+### Push Delivery {#common-receiver-push}
+Receivers MUST be able to accept events using the Push-Based Security Event Token (SET) Delivery Using HTTP {{RFC8935}} specification.
+
+### Implicitly Added Subjects {#common-receiver-subjects}
+Receivers MUST assume that all subjects are implicitly included in a Stream, without any `AddSubject` method invocations.
+
+## Event Subjects {#common-event-subjects}
+The following subject identifier formats from "Subject Identifiers for Security Event Tokens" {{RFC9493}} MUST be supported:
+
+* `email`
+* `iss_sub`
+
+Receivers MUST be prepared to accept events with any of the subject identifier formats specified in this section. Transmitters MUST be able to send events with at least one of subject identifier formats specified in this section.
```

Comment:
Can we rephrase this as "at least one" for the receiver as well, otherwise it adds more burden on the receiver than the transmitter

Reply:
Having "at least one" on both sides can lead to incompatibilities, so for interoperability we can require both sides to support both subject types or only one side to support "at least one"